### ISSUE TITLE
Segment filter "email read count equals 0" not returning expected result

### ISSUE THREAD

**Issue by rowlandhill** [mautic/mautic#5041]
What type of report is this:

| Q  | A
| ---| ---
| Bug report? | Yes
| Feature request? | No
| Enhancement? | No

## Description:
A customer asked if they could create a Segment to see which customers have not opened a single email in the last 6 months.  I began with just filtering out customers who haven't opened an email at all.  I selected "Email Read Count" and set it equal to "0", but the Segment has 0 contacts, even though I know there are multiple contacts who have not read emails within my instance.  Instance url and Segment number can be provided if necessary.

## If a bug:

| Q   | A
| --- | ---
| Mautic version | Yes
| PHP version | 

### Steps to reproduce:
1. Go to Segments
2. Select filter "Email Read Count"
3. Set "equals" "0"
 
### Log errors: 

_Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable._ 

@cceruti

**Comment by kuzmany** [comment]
Bugfix #5043

**Comment by mqueme** [comment]
This issue has not been resolved fully, the combination will only return contacts that have never opened an email before.

**Comment by rowlandhill** [comment]
This worked for me in my account, info.rowlandhill.co, which is on release 2018.10.05-8

These are the Segments I created:

https://info.rowlandhill.co/s/segments/view/170

This shows all Contacts who have never read an Email.  Filter is Email Read Count equals 0.  This filled with 120 Contacts.  I checked all the Contacts in the Segment with valid email addresses and confirmed they had not read an email.

I also created this Segment: 

https://info.rowlandhill.co/s/segments/view/171

This shows all Contacts who have read at least 1 email.  Filter is Email Read Count greater than 0.  There are 21 Contacts in here and all have at least 1 Email read. This filled with 21 Contacts, which brings the total between the two to 141, and that's how many known Contacts I have in my Account.

This looks good to me, @escopecz !
